content/en/docs/userguide/try_authorization.md

```diff
@@ -0,0 +1,226 @@
+    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                   Dload  Upload   Total   Spent    Left  Speed
+    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0{
+    "headers": {
+      "Accept": "*/*", 
+      "Host": "httpbin:8000", 
+      "User-Agent": "curl/8.5.0"
+    }
+  }
+  100   105  100   105    0     0  18078      0 --:--:-- --:--:-- --:--:-- 21000
+  ```
+
+  
+
+- Verify if requests from other IPs are being denied.
+
+  ```
+  kubectl exec sleep-78ff5975c6-phhll -c sleep -- curl  http://httpbin:8000/headers
+    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                   Dload  Upload   Total   Spent    Left  Speed
+    0   105    0     0    0     0      0      0 --:--:--  0:01:00 --:--:--     0
+  curl: (56) Recv failure: Connection reset by peer
+  ```
+
+  
+
+- Clean up the AuthorizationPolicy.
+
+  ```log
+  kubectl delete AuthorizationPolicy allow-by-srcip -n default
+  ```
+
+  
+
+## Configure DENY authorization policy
+
+- Create a "deny-by-srcip" authorization policy for the httpbin workload within the specified namespace, which denies requests from a particular IP address, execute the following command
+
+  ```
+  kubectl apply -f - <<EOF
+  apiVersion: security.istio.io/v1beta1
+  kind: AuthorizationPolicy
+  metadata:
+   name: deny-by-srcip
+   namespace: default
+  spec:
+   selector:
+     matchLabels:
+       app: httpbin
+   action: DENY
+   rules:
+   - from:
+     - source:
+         ipBlocks:
+         - 10.244.1.46/32
+  EOF
+  ```
+
+- Verify whether requests from the corresponding IP are being denied.
+
+  ```
+  kubectl exec sleep-78ff5975c6-plh7r -c sleep -- curl  "http://httpbin:8000/headers"
+    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                   Dload  Upload   Total   Spent    Left  Speed
+    0   105    0     0    0     0      0      0 --:--:--  0:01:04 --:--:--     0
+  curl: (56) Recv failure: Connection reset by peer
+  ```
+
+- Verify if requests from other IPs are being allowed.
+
+  ```
+  kubectl exec sleep-78ff5975c6-phhll -c sleep -- curl  "http://httpbin:8000/headers"
+    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                   Dload  Upload   Total   Spent    Left  Speed
+    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0{
+    "headers": {
+      "Accept": "*/*", 
+      "Host": "httpbin:8000", 
+      "User-Agent": "curl/8.5.0"
+    }
+  }
+  100   105  100   105    0     0  11284      0 --:--:-- --:--:-- --:--:-- 11666
+  ```
+
+  
+
+- Clean up the AuthorizationPolicy.
+
+  ```log
+  kubectl delete AuthorizationPolicy deny-by-srcip -n default
+  ```
+
+  
+
+## Clean up
+
+Please refer [cleanup](https://kmesh.net/en/docs/setup/quickstart/#clean-up)
+
+
+
+## AuthorizationPolicy
```

Comment:
do we need to copy the upstream api here, why not link to upstream

Reply:
> do we need to copy the upstream api here, why not link to upstream

The upstream API document seems to include L4-L7  all capabilities

Reply:
Make sense, i missed that